### PR TITLE
give the vue dev tool the ablity to select the correspond component w…

### PR DESCRIPTION
**Pull request by zzuligy**
provide vue dev tool the ablity to select the the correspond component when select a page element  by inspector.
when the page has lots of component,this function can let the develop find the correspond compoent more fastly.

### CODE REVIEW

File: src/backend/index.js [hunk 1]
```diff
@@ -59,17 +59,46 @@ function connect () {
...
+      let instanceContainer = []
+      instanceMap.forEach((instance, id) => {
+        let curSelectedDom = document.querySelector(`[data-v_track_id="${uuid}"]`)
+        let isAncestorInstance = instance.$el.contains(curSelectedDom)
+        if (isAncestorInstance) {
+          instanceContainer.push(instance)
+        }
+      })
+      //sort ancestor
+      instanceContainer.sort(function (m, n) {
```
Comment (posva) [inline src/backend/index.js]: What is the sort for here?

**Comment by posva** [comment]
do you mean #479 ?

**Comment by zzuligy** [comment]
yes.we do the same thing.but his code is more elegant.for example the contextMenu,the __vue__, the exapnd of the selected component

**Comment by Akryum** [comment]
I guess we will go with #476 and #479.

**Pull request closed without merging**